js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -263,6 +264,20 @@ export class MiniAppBridge {
     });
   }
 
+  /**
+   * Get access token from native hostapp.
+   */
+  getAccessToken() {
+    return new Promise<TokenData>((resolve, reject) => {
+      return this.executor.exec(
+        'getAccessToken',
+        null,
+        tokenData => resolve(JSON.parse(tokenData) as TokenData),
```

Comment:
We send the JSON string from the respective Android/iOS SDK's, do we still need to `JSON.parse` ?

https://github.com/rakutentech/ios-miniapp/pull/161/files#diff-fd17266a8049bc8366e4f1fba97d7a5bc0568eb62b4d3a2bf10122daefbe90deR211

Reply:
Yeah we need it.
The native sdk sends the json string and the js sdk parse that json string into object. The object here is `TokenData`.